tests/core/test_circuit_breaker.py

```
@@ -0,0 +1,199 @@
+"""
+Unit tests for circuit breaker implementation.
+"""
+import pytest
+import asyncio
+import time
```

Ruff (0.8.2)

`time` imported but unused

Remove unused import: `time`

(F401)